packages/nimble-components/src/anchor-tabs/tests/anchor-tabs.stories.ts

```diff
@@ -0,0 +1,83 @@
+import { html, repeat, when } from '@microsoft/fast-element';
+import type { Meta, StoryObj } from '@storybook/html';
+import { withXD } from 'storybook-addon-xd-designs';
+import { createUserSelectedThemeStory } from '../../utilities/tests/storybook';
+import '../../all-components';
+
+interface TabsArgs {
+    tabs: TabArgs[];
+    activeId: string;
+    toolbar: string;
+}
+
+interface TabArgs {
+    label: string;
+    id: string;
+    href: string;
+    disabled: boolean;
+}
+
+const overviewText = `Per [W3C](https://w3c.github.io/aria-practices/#tabpanel) - Tabs are a set of layered
```

Comment:
Looks like this is identical to the text for tabs. Could you:
1. update it to describe what anchor tabs are and how they differ from regular tabs
2. also update the tabs documentation to mention anchor tabs and describe when to use each

Reply:
Good catch. I had meant to come back to that. I've updated them both now.